SpatialGDK/SpatialGDK.uplugin

```diff
@@ -1,13 +1,13 @@
 {
   "FileVersion": 3,
   "Version": 5,
-  "VersionName": "0.8.0",
+  "VersionName": "0.8.1",
```

Comment:
0.9 ?

Reply:
Do we usually update the version before we release?

Reply:
You ask wise questions